app/components/user-page/course-progress-list-item.js

```diff
@@ -15,7 +15,15 @@ export default class CourseProgressListItemComponent extends Component {
   }
 
   get completedStagesCount() {
```

Comment:
Let's handle de-duplication in the API - this should just be `this.completedStageSlugs.length`